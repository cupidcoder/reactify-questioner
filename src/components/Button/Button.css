.Button {
  background-color: #305b8c;
  color: #ffffff;
  border: none;
  padding: 1rem 2rem;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  margin: 4px 2px;
  cursor: pointer;
}