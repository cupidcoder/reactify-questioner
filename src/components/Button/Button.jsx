import React from 'react';
import style from './Button.css';

const Button = ({ onClick, children }) => (
  <button onClick={onClick} className={style.Button} type='button'>
    {children}
  </button>
);

export default Button;
