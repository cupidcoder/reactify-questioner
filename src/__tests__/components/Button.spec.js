import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../components/Button/Button';

describe('Button', () => {
  it('Should correctly save snapshot of button', () => {
    const ButtonWrapper = shallow(<Button />);
    expect(ButtonWrapper).toMatchSnapshot();
  });
});
