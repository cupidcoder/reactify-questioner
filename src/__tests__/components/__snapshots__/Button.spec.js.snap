// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First test Should correctly save snapshot of button 1`] = `ShallowWrapper {}`;
